WooCommercePOS/WooCommercePOS/Utils/Color+WooCommercePOS.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+
+extension Color {
```

Comment:
I will keep updating this file but just wanted to get this PR as soon as possible so we can update it as we go.